addons/im_livechat/static/src/legacy/public_livechat.js

```diff
@@ -187,6 +188,18 @@ var LivechatButton = Widget.extend({
                     channel_uuid: this._livechat.getUUID(),
                     page_history: history,
                 });
+            } else if (notification[1].info === 'typing_status') {
+                const partnerID = notification[1].partner_id;
+                if (partnerID !== this._livechat.getOperatorPID()[0]) {
+                    // ignore typing display in case typing is on current
+                    // partner (= non-operator).
+                    return;
+                }
```

Comment:
try to send current partner id in `get_livechat_info` so that we can properly compare it here